run-container.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
 
-docker run -ti --rm --name rhoas-devsandbox --entrypoint /bin/bash ddoyle/rhoas-devsandbox-workshop-tools
+registry=${REGISTRY:-quay.io}
```

Comment:
Do we need that script? Having this in readme could be enough?

Reply:
No, I just did not want to remove it in case it is used